config/config.exs

```diff
@@ -39,6 +39,9 @@ config :elixir_analyzer,
     "chessboard" => %{
       analyzer_module: ElixirAnalyzer.TestSuite.Chessboard
     },
+    "dna-encoding" => %{
+      analyzer_module: ElixirAnalyzer.TestSuite.DNA
```

Comment:
Could you call the module `DnaEncoding`? Or `DNAEncoding`? The capitalization is a bit awkward but we need the `Encoding` for consistency.

Reply:
I was not sure about this, as the module of the exercise is named `DNA` as you can see [here](https://github.com/exercism/elixir/blob/d00ca743340fcc328c70ee351274f91b57fd4c8d/exercises/concept/dna-encoding/lib/dna.ex#L1) and I thought that the anaylzer always refered to an existing module :thinking: (but the project/directory/exercise is named `dna-encoding`).

Should I leave `DNA` or rename it to `DNAEncoding`?

Reply:
The exercise itself can break convention, but here we are referencing many exercises, so I think I would prefer having the full name.